packages/parser/test/specs/object-source/object-source.test.ts

```diff
@@ -1,33 +1,18 @@
-import type { ValidAPIDefinition } from '../../utils/helper.js';
+import type { ValidAPIDefinition } from '../../utils.js';
 
 import { describe, it, expect } from 'vitest';
 
-import { OpenAPIParser } from '../../../src/index.js';
-import * as path from '../../utils/path.js';
+import { dereference, bundle } from '../../../src/index.js';
 
 import bundledAPI from './bundled.js';
 import dereferencedAPI from './dereferenced.js';
 import parsedAPI from './parsed.js';
 
 describe('Object sources (instead of file paths)', () => {
   it('should dereference an object that references external files', async () => {
-    const parser = new OpenAPIParser<ValidAPIDefinition>();
-    const api = await parser.dereference(structuredClone(parsedAPI.api));
+    const api = await dereference<ValidAPIDefinition>(structuredClone(parsedAPI.api));
 
-    expect(api).to.equal(parser.schema);
     expect(api).to.deep.equal(dereferencedAPI);
-
-    // The API path should be the current directory, and all other paths should be absolute
-    const expectedPaths = [
-      path.cwd(),
-      path.abs('specs/object-source/definitions/definitions.json'),
-      path.abs('specs/object-source/definitions/name.yaml'),
-      path.abs('specs/object-source/definitions/required-string.yaml'),
-    ];
-    expect(parser.$refs.paths()).to.have.same.members(expectedPaths);
```

Comment:
Our parser no longer exposes the `$refs` property that `json-schema-ref-parser` has so I've removed these assertions. That `$refs` stuff is well tested within `json-schema-ref-parser` already.